how-we-work-together.md

```diff
@@ -174,69 +174,69 @@ Many Sparkboxers are involved in initial staffing, budget, and timeline conversa
 
-* Present design concepts to the client with a detailed explanation and reasons behind visual choices
+* Present design concepts to the client with a detailed explanation of reasons behind visual choices
 
     * Talk about the why, not the what
 
     * Reiterate the visual vocabulary
 
-    * Discuss how the presented concept will help to accomplish the project goals
+    * Discuss how the presented concept will help accomplish the project goals
 
-    * If the client doesn’t agree with our rational, we listen and take every effort to address their concerns
+    * If the client doesn’t agree with our rational, we listen and make every effort to address their concerns
 
-    * We believe it is acceptable and our job to respectfully push back when we feel strongly that a client suggestion is detrimental to the project, and give reasons why
+    * We believe it is acceptable and our job to respectfully push back and give reasons why when we feel strongly that a client suggestion is detrimental to the project
 
-* Communicate/educate the client that end goal is browser
+* Communicate to/educate the client that the end goal is browser
 
-    * Different browsers render fonts, colors differently
+    * Different browsers render fonts and colors differently
 
     * May not show comps of every page
 
-### We Translate Concepts to Frontend Code 
+### We Translate Concepts to Frontend Code
 
 * Ensure that the product is polished and refined before launch
 
-* Transfer ownership of assets 
+* Transfer ownership of assets
 
-    * fonts, photography and help client make necessary typography and image purchases
+    * fonts, photography (help client make necessary typography and image purchases)
 
 * Build with accessibility and performance in mind
 
-* Design modularly – keep reuse of code in mind when needed
+* Design modularly—keep reuse of code in mind when needed
 
 * Be available and involved in UI decomps to communicate design intent is carried through
 
-* As project progresses, provide alternate solutions to areas that can’t be built in time/budget
+* As the project progresses, provide alternate solutions to areas that can’t be built in time/budget
 
-* Help communicate the experience strategy with well articulated design concept notes 
+* Help communicate the experience strategy with well articulated design concept notes
 
 * Hit deadlines set for the project
 
-* Help PMs know level of effort for outstanding Github issues, and break into multiple cards when necessary
+* Help PMs know the level of effort required for outstanding Github issues and break issues into multiple cards when necessary
 
 * Predict and communicate what will be hard to build and have a backup plan
 
-* Know when to compromise design where needed, know when to push back
+* Know when to compromise design where needed and know when to push back
 
 ### We Share Design Learnings and Embrace Improvement

```

Comment:
Two of the points under this heading are

* File organization/documentation
* Accountability on side projects

All of the other bullet points under here are actions (provide, give, actively participate), but these are not. I'm not sure how these relate to sharing learnings and embracing improvement. Do we clearly write documentation and organize out files? And provide accountability on side projects? For these to be clear, I just think they need to be turned into actions so people know how this is educating and improving.